routes/users.js

```diff
@@ -66,6 +68,23 @@ router.delete('/:userId', async (req, res) => {
     res.end();
 });
 
+router.post('/login', async (req, res) => {
+    const body = req.body;
+    const username = body.username;
+    const password = body.password;
+    const user = await userService.findByUserNameAndPassword(username, password);
+    if (user.length === 1) {
+        const payload = { sub: user.age, title: user.username };
+        const token = jwt.sign(payload, secret, { expiresIn: 120 });
+        res.send(token);
+    } else {
+        res.status(401).send({
+            success: false,
+            message: 'Bad username/password combination'
+        });
+    }
+});
+
```

Comment:
It's better to keep login separately, as it will help you to avoid an if statement in your auth middleware.